index.bs

```diff
@@ -7063,11 +7059,21 @@ an |ownership type|, a |serialization internal map|, a |realm| and a |session|:
         production in the [=local end definition=], with the <code>handle</code>
         property set to |handle id| if it's not null, or omitted otherwise.
 
+    <dt>|value| has a \[[GeneratorState]] [=internal slot=]
```

Comment:
Similar to my other comment regarding async iterators before, it doesn't seem like this will work for `AsyncGenerator` instances given that those have a [`AsyncGeneratorState` property](https://tc39.es/ecma262/#sec-properties-of-asyncgenerator-intances).

Reply:
Yeah, I think you're right, I've added the check for `AsyncGeneratorState` internal slot